website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md

```diff
@@ -19,162 +19,185 @@ limitations under the License.
 
 # Java multi-language pipelines quickstart
```

Comment:
I think we should call this "Java multi-language quickstart for using Python transforms" and change the title/URL etc. accordingly.

Reply:
I would push back on this a little bit. The intro makes it clear what we're doing in this quickstart: "In the examples below, the multi-language pipeline is built with the Beam Java SDK, and the cross-language transform is built with the Beam Python SDK." It's analogous to what we're doing with the Python multi-lang quickstart. At some point in the future, we _might_ need to change both sets of title/URL and narrow the focus, but we're not there yet. I'd rather maintain the symmetry. But let me know...

Reply:
Sounds good. Thanks.